working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
```

Comment:
Might it be more clear to annotate the "edit" access? I feel like the default assumption is that open is "viewable", so it's the "edit" escalation that's more clear to see noted :)

Reply:
Or we could just be explicit, and have a notation for both?

👀 view-only
📝 view & edit

Reply:
> at any point
Does this mean "for any reason" as well?

Reply:
I like the explicit double notation @patcon! will add in

Reply:
> > at any point
> 
> Does this mean "for any reason" as well?

@uditvira -- I'm not sure I'm seeing the crux of the difference here. Could you say more? 

I thought this captured the conversation we had: currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured.

Just to restate my vibes from the convo: in time we should revisit and set norms of what stays open + a more explicit consent process (to steal @patcon's wording) for the things that we've noticed as the sites where we need differential shading. I think over time we'll get to a point where we recognize when that happens. As we learn I think we'll need to have some discussion and reflection about the reasons we are okay with around privacy.

Reply:
> at any point

... in time?

I assumed you were referring to an occasion (before, during, or after), which does not describe the scope of and limits to redactions

> currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured

This is the clarification I was looking for, thanks. Yes, I agree there is no need to systematize more explicitly for now

Reply:
Okay. If additional words would make things more clear @uditvira please propose a friendly amendment!